.travis.yml

```diff
@@ -7,13 +7,14 @@ php:
     - '7.2'
     - '7.3'
     - '7.4'
+    - '8.0'
 
 before_install:
     - mkdir -p build/logs
 
 install:
     - composer self-update --ansi --no-interaction
-    - composer install --ansi --no-interaction --no-suggest
```

Comment:
is that just compatibility with composer v2 in order to get rid of the [deprecation](https://php.watch/articles/composer-2#no--no-suggest) message?

Reply:
Yes, and if you have lots of packages in a project, the suggests output can be so noisy it loses its value.